plugins/bulk-import/src/components/Repositories/EditCatalogInfo.tsx

```diff
@@ -34,10 +34,14 @@ const EditCatalogInfo = ({
   const bulkImportApi = useApi(bulkImportApiRef);
   const { setSubmitting, setStatus, isSubmitting } =
     useFormikContext<AddRepositoriesFormValues>();
-
-  const yamlContent = yaml.loadAll(
-    importStatus?.github?.pullRequest?.catalogInfoContent,
-  )[0] as Entity;
+  let yamlContent = {} as Entity;
+  try {
+    yamlContent = yaml.loadAll(
+      importStatus?.github?.pullRequest?.catalogInfoContent,
+    )[0] as Entity;
+  } catch (e) {
```

Comment:
should we log the error or show to user?

Reply:
Added `console.warn(e)`